frontend/src/libs/components/input/input.tsx

```diff
@@ -20,6 +20,7 @@ type Properties<T extends FormFieldValues> = {
   maxLength?: number;
   autoComplete?: 'off' | 'on';
   isChatInput?: boolean;
+  disabled?: boolean;
```

Comment:
Prefer the acceptance criteria rule here

Reply:
I don't think I fully understand what you mean here, can you please provide me a little more context so I can understand what is required?

Reply:
@v0ldemar01

Reply:
I mean B19 rule of quality criteria